packages/components/src/DataTable/Footer.css

```diff
@@ -0,0 +1,48 @@
+.pagination {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  min-height: calc(var(--space-smaller) + var(--space-largest));
+  padding: var(--space-small) var(--space-base);
+  border-top: var(--border-base) solid var(--color-border);
+  flex-wrap: wrap;
+  gap: var(--space-small);
+}
+
+.paginationInfo {
+  flex: 9999;
```

Comment:
Why do we need this `9999`? Isn't `1` enough to take the whole space? 🤔

Reply:
I needed to add this to force the other flex 1 container to not grow when the screen gets bigger but still occupies the available width when the parent container wraps. Solving [this](https://github.com/GetJobber/atlantis/pull/925#issuecomment-1294076257)

Reply:
✅